packages/design-system/src/components/InlineError/InlineError.tsx

```diff
@@ -9,24 +11,31 @@ import classNames from 'classnames';
 interface InlineErrorProps {
   children?: React.ReactNode;
   className?: string;
+  displayErrorIcon?: boolean;
   id?: string;
   inversed?: boolean;
 }
 
 export function InlineError({
   children,
   className,
+  displayErrorIcon,
   id,
   inversed,
 }: InlineErrorProps): React.ReactElement {
+  getInlineErrorIconDisplay() ? (displayErrorIcon = true) : (displayErrorIcon = false);
```

Comment:
[fix] with this ternary, if another component uses `<InlineError>` and passes the prop `displayErrorIcon`, the value of the prop will never get used, it will always be overwritten by the flag.

Consider doing either one of the following
1. Not make `displayErrorIcon` a prop and always use the value of the flag like `const displayErrorIcon = getInlineErrorIconDisplay()`
2. Update this line to be more of an "order of precedence" like use the value of the prop, if that doesn't exist use the flag. Something like `const displayIcon = displayErrorIcon || getInlineErrorIconDisplay()`